data/items.ts

```diff
@@ -1587,7 +1587,8 @@ export const Items: import('../sim/dex-items').ItemDataTable = {
 					for (const pokemon of this.getAllActive()) {
 						if (pokemon.switchFlag === true) return;
 					}
-					if (target.useItem()) target.switchFlag = true;
+					target.switchFlag = true; // hack
```

Comment:
```suggestion
					target.switchFlag = true; // Symbiosis relies on this being set here
```
(also for eject button of course)